src/parameters/all_parameters.cpp

```diff
@@ -272,17 +288,23 @@ void AllParameters::parse_parameters (dealii::ParameterHandler &prm)
     use_weak_form = prm.get_bool("use_weak_form");
     use_collocated_nodes = prm.get_bool("use_collocated_nodes");
     use_split_form = prm.get_bool("use_split_form");
+    use_curvilinear_split_form = prm.get_bool("use_curvilinear_split_form");
+    use_weight_adjusted_mass = prm.get_bool("use_weight_adjusted_mass");
     use_periodic_bc = prm.get_bool("use_periodic_bc");
     use_energy = prm.get_bool("use_energy");
     use_L2_norm = prm.get_bool("use_L2_norm");
-    use_classical_FR = prm.get_bool("use_classical_Flux_Reconstruction");
+    use_classical_FR = prm.get_bool("use_classical_FR");
     sipg_penalty_factor = prm.get_double("sipg_penalty_factor");
 
     const std::string conv_num_flux_string = prm.get("conv_num_flux");
-    if (conv_num_flux_string == "lax_friedrichs") conv_num_flux_type = lax_friedrichs;
-    if (conv_num_flux_string == "split_form")     conv_num_flux_type = split_form;
-    if (conv_num_flux_string == "roe")            conv_num_flux_type = roe;
-    if (conv_num_flux_string == "l2roe")            conv_num_flux_type = l2roe;
+    if (conv_num_flux_string == "lax_friedrichs")                                               conv_num_flux_type = lax_friedrichs;
+    if (conv_num_flux_string == "split_form")                                                   conv_num_flux_type = split_form;
+    if (conv_num_flux_string == "roe")                                                          conv_num_flux_type = roe;
+                                                                                                 
+    if (conv_num_flux_string == "l2roe")                                                        conv_num_flux_type = l2roe;
+    if (conv_num_flux_string == "central_flux")                                                 conv_num_flux_type = central_flux;
+    if (conv_num_flux_string == "entropy_conserving_flux" && pde_string == "burgers_inviscid" ) conv_num_flux_type = entropy_cons_flux;
```

Comment:
You need to make it fail if entropy_conserving_flux is chosen, but it is not a Burger's PDE. Otherwise, this is a silent failure, and no one will ever know why it fails unless they know about this line.

Reply:
fixed

Reply:
Fix the typos in the message please.

Reply:
fixed